### ISSUE TITLE
Need trending song and artist

### ISSUE THREAD

**Issue by MohammedShameer1532** [sumitkolhe/jiosaavn-api#114]
We need trending songs and trending artists to complete the API

**Comment by Nikhil-sha** [comment]
Yes! I also wanted to say the same. This will be a very good feature which will give a great experience to the user and will help in finding new songs and artists.
